Libplanet/Blockchain/BlockChain.cs

```diff
@@ -638,6 +638,20 @@ bool renderActions
             }
         }
 
+        /// <summary>
+        /// Render actions from next block of <paramref name="from"/>.
+        /// </summary>
+        /// <param name="from">Block to start rendering.</param>
+        internal void Render(Block<T> from)
```

Comment:
Did you declare this method to use `BlockChain<T>.IterateBlocks()` method from `Swarm<T>`?

Reply:
Currently, yes.